lib/trace/index.js

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
This file definitely has more of a "server" element to it. That's obvious in that you're setting up the endpoint handlers. Existing convention is for handlers to live in `server/`. Handlers shouldn't have a whole lot of meet in them, other than validating request parameters, delegating to a 'controller' (aka some module in `lib/trace`) and then crafting a tchannel response from the results of the controller.